parser-typechecker/src/Unison/Server/Types.hs

```diff
@@ -97,6 +106,67 @@ data DefinitionDisplayResults =
     , missingDefinitions :: [HashQualifiedName]
     } deriving (Eq, Show, Generic)
 
+data TermTag = Doc | Test
```

Comment:
These types were moved here from endpoint-specific files since they get used in more than one place.